x-pack/plugins/reporting/server/lib/store/store.ts

```diff
@@ -89,7 +108,7 @@ export class ReportingStore {
       id: report._id,
       body: {
         ...report.toEsDocsJSON()._source,
-        process_expiration: new Date(0), // use epoch so the job query works
+        process_expiration: null,
```

Comment:
A quirk of ESQueue allowed new jobs to be created with the epoch as the process expiration time.